README.md

```diff
@@ -1,58 +1,52 @@
-<h1 align="center">Dat Desktop</h1>
-
-<div align="center">
-  <strong>Peer to peer data sharing app built for humans.</strong>
-</div>
-
-<br />
-
-<div align="center">
-  <!-- Build Status -->
-  <a href="https://travis-ci.org/dat-land/dat-desktop">
-    <img src="https://img.shields.io/travis/dat-land/dat-desktop/master.svg?style=flat-square"
-      alt="Build Status" />
-  </a>
-  <!-- Standard -->
-  <a href="https://standardjs.com">
-    <img src="https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat-square"
-      alt="Standard" />
-  </a>
-</div>
+# Dat Desktop
+
+**Peer to peer data sharing app built for humans.**
+
+[!["Build Status"](https://img.shields.io/travis/dat-land/dat-desktop/master.svg?style=flat-square)](https://travis-ci.org/dat-land/dat-desktop) 
+[!["Standard](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat-square)](https://standardjs.com)
 
 ![screenshot](assets/screenshot.png)
 
 ## Table of Content
+
 - [Download](#download)
 - [Commands](#commands)
 - [FAQ](#faq)
 - [License](#license)
 
 ## Download
 
-To run the app locally check out our [download
-guide](https://datproject.org/install) or install directly from the command
-line using [Homebrew Cask](https://caskroom.github.io). Currently only MacOS is
-supported.
+_Dat_ needs to be installed first. To run the app locally check out our [download
+guide](https://datproject.org/install) or install directly from the command line:
```

Comment:
I never tried installing it through homebrew-cask. Do we have to do something to update them? Should we create issues to deploy DAT for debian/windows-store/mac-store/etc. ?